<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>dlgDemo02</class>
	<widget class="QDialog" name="dlgDemo02">
		<property name="geometry">
			<rect>
			<x>0</x>
			<y>0</y>
			<width>400</width>
			<height>150</height>
			</rect>
		</property>
        <property name="windowTitle">
			<string>Demo 02: Creacion de Controles en una UI sin disenador</string>
		</property>
        <widget class="QLabel" name="lblNombre">
			<property name="geometry">
				<rect>
				 <x>30</x>
				 <y>20</y>
				 <width>150</width>
				 <height>16</height>
				</rect>
			</property>
			<property name="text">
				<string>Ingresa el Nombre:</string>
			</property>
		</widget>
        <widget class="QLineEdit" name="txtNombre">
			<property name="geometry">
				<rect>
				 <x>180</x>
				 <y>20</y>
				 <width>150</width>
				 <height>16</height>
				</rect>
			</property>			
		</widget>
        <widget class="QPushButton" name="btnAceptar">
			<property name="geometry">
				<rect>
				 <x>30</x>
				 <y>70</y>
				 <width>100</width>
				 <height>30</height>
				</rect>
			</property>
			<property name="text">
				<string>Aceptar</string>
			</property>
		</widget>
        <widget class="QPushButton" name="btnCancelar">
			<property name="geometry">
				<rect>
				 <x>280</x>
				 <y>70</y>
				 <width>100</width>
				 <height>30</height>
				</rect>
			</property>
			<property name="text">
				<string>Cancelar</string>
			</property>
		</widget>
	</widget>
</ui>